PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,9 +8,9 @@ The build axis can be controlled by prefixing a ! on the following as appropriat
 
 CORE TOMCAT AS_TESTS RTS JACOCO XTS QA_JTA QA_JTS_OPENJDKORB PERFORMANCE LRA DB_TESTS mysql db2 postgres oracle
 
-By default the pull request will run with `JDK11`, if you prefix a `!` to `JDK11` then it will not run with `JDK11`.
+By default the pull request will run with `JDK11` and `JDK17`, if you prefix a `!` to `JDK11` then it will not run with `JDK11`.
 
-By default the PR will not compile and run with JDK 17. To override this you need to concatenate the two sets of characters: `JDK` `17` as a single word and provide the result into the description.
+By default the PR will not compile and run with JDK 21. To override this you need to concatenate the two sets of characters: `JDK` `21` as a single word and provide the result into the description.
```

Comment:
These paragraphs are clunky and awkward to read. I'd suggest the following instead:

> Pull requests build and run with `JDK11` and `JDK17`. Add the prefix `!` to disable these defaults. Include the text JDK21 to build and run with JDK21.

Reply:
Thanks Mike, updated!